setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
Lots of packages usually import stuff in `__init__.py` as convenience imports though, so people can `from yolapy import client, model, potato` instead of explicitly importing them from each submodule, which also makes backward compatibility easier if we wanna move code around.